content/en/docs/reference/networking/virtual-ips.md

```diff
@@ -133,7 +133,7 @@ iptables:
 
 ##### Performance optimization for `iptables` mode {#minimize-iptables-restore}
```

Comment:
The sense could change here; for example:
```markdown
##### Legacy `minSyncPeriod` configuration in `iptables` mode {#minimize-iptables-restore}
```

Also, consider moving this into or _after_ the `minSyncPeriod` section.

Reply:
Perhaps this heading isn't needed at all?